src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/Vedtaksperioder.tsx

```diff
@@ -40,10 +43,13 @@ export const Vedtaksperioder: React.FC<Props> = ({
     settVedtaksperioderFeil,
 }) => {
     const { erStegRedigerbart } = useSteg();
-    const { settUlagretKomponent } = useApp();
+    const { request, settUlagretKomponent } = useApp();
+    const { behandling } = useBehandling();
 
     const [idNyeRader, settIdNyeRader] = useState<Set<string>>(new Set());
 
+    const [foreslåPeriodeFeil, settForeslåPeriodeFeil] = useState<string | undefined>();
```

Comment:
Her trenger man ikke å sette `string | undefined`, då `useState<string>()` uten å initiere med default verdi automatisk blir `string | undefined`
```suggestion
    const [foreslåPeriodeFeil, settForeslåPeriodeFeil] = useState<string>();
```